src/phalanx/testing/expandcharts.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
I am mildly against putting test suite code into the actual Python package for something, so this part of the change feels weird to me and I'd rather keep things that are only useful for testing in the `tests` directory. I can't imagine anyone wanting to run that script manually; it's just a hack to make `helm lint` sort of work and there's probably some better way to do that. That said, just a mild objection, and more on a matter of aesthetics than any actual issue, so do feel free to decide you don't agree.

Reply:
Ah I see. My thinking was that the phalanx's python package job was to provide test, documentation build, and development templating support code for phalanx rather than "be phalanx", hence moving the expandcharts script into the phalanx package so that down the line we can use our standard infrastructure for testing that code and providing dependencies for it.

So this isn't test suite code for the phalanx python package, but functionality of the phalanx Python package.

Does that kind of make sense? If not I'm happy to revert.

Reply:
Oh! I see. That's consistent with the docs, yes. That makes more sense. Okay, I'm convinced.

I was thinking of the phalanx package as primarily being the installer, so had a more limited view of it.

I think we probably should think about whether we want to keep the GitPython dependency out of the main dependencies for the eventual use of the phalanx package as the installer for people who just want to install Phalanx and not test it, but we can also cross that bridge later, and use some other optional dependency name for it.

Reply:
I'm glad we're on the same page. Yeah, we can be more specific with the dependencies... we can work that out when get cross that bridge with a more developer/user focus.